tests/Egret.Tests/Serialization/Audacity/AudacityImporterTests.cs

```diff
@@ -0,0 +1,193 @@
+                i.TimeEnd.AlmostEqual(7.2446258503) &&
+                i.SelLow.AlmostEqual(1) &&
+                i.SelHigh.AlmostEqual(10)
+            );
+            result.Tracks.First().Labels.Skip(1).First().Should().Match<Label>(i =>
+                i.Title == "test 3" &&
+                i.TimeStart.AlmostEqual(15.9714285714) &&
+                i.TimeEnd.AlmostEqual(24.44) &&
+                i.SelLow.AlmostEqual(10) &&
+                i.SelHigh.AlmostEqual(10000)
+            );
+
+            result.Tracks.Skip(1).First().Should().Match<LabelTrack>(i =>
+                i.Name == "Track 1" &&
+                i.IsSelected == 1 &&
+                i.Height == 90 &&
+                i.Minimized == 0 &&
+                i.Labels.Count == 1
+            );
+            result.Tracks.Skip(1).First().Labels.First().Should().Match<Label>(i =>
+                i.Title == "test 2" &&
+                i.TimeStart.AlmostEqual(8.0228571429) &&
+                i.TimeEnd.AlmostEqual(18.98) &&
+                i.SelLow == 0 &&
+                i.SelHigh == 0
+            );
+        }
+        
+        public static readonly TestFile HostConfig = ("/abc/host.egret.yaml", @"
+test_suites:
+  host_suite:
+    tests:
+      - file: bird*.wav
+    include_tests:
+      - from: /abc/example1.aup
+");
+
+        public static readonly TestFile GuestConfig = (@"/abc/example1.aup", File.ReadAllText(ExampleProjectFile1));
+        
+        [Fact]
+        public async Task TestConfigDeserializer()
+        {
+            TestFiles.AddFile("/abc/bird1.wav", "");
+            TestFiles.AddFile("/abc/bird2.wav", "");
+            TestFiles.AddFile("/def/bird3.wav", "");
+            TestFiles.AddFile("/def/bird4.wav", "");
+            TestFiles.AddFile(HostConfig);
+            TestFiles.AddFile(GuestConfig);
+            
+            (Config config, Seq<Error> errors) = await this.configDeserializer.Deserialize(
+                TestFiles.FileInfo.FromFileName(HostConfig.Path)
+            );
+
+            errors.Should().BeEmpty();
+            config.Should().NotBeNull();
+
+            config.TestSuites.Should().HaveCount(1);
+            
+            var testSuit = config.TestSuites["host_suite"];
+            testSuit.IncludeTests.ToList().Should().HaveCount(1);
+
+            var includeTests = testSuit.IncludeTests[0];
+            includeTests.From.Should().Be(@"/abc/example1.aup");
+
+            var testsCases = includeTests.Tests;
+            testsCases.ToList().Should().HaveCount(1);
+
+            var tests = testsCases[0].Expect;
+
+            tests[0].Should().BeOfType<BoundedExpectation>();
+            tests[1].Should().BeOfType<BoundedExpectation>();
+            tests[2].Should().BeOfType<TemporalExpectation>();
+
+            var temporalTolerance = includeTests.TemporalTolerance ?? 0.5;
+            var spectralTolerance = includeTests.SpectralTolerance ?? 0.5;
+
+            tests.ForEach((e, index) =>
```

Comment:
```suggestion
            expectations.ForEach((e, index) =>
```

suite: many files
test: one file + some expectations
expectation: one event that we expect with some assertions
assertion: checking one individual attribute of the event